### PR TITLE
Feature/a11y fixes

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/app/components/shared/SkipLinks/SkipLink.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components"
```
Comment (gilleswittenberg) [inline src/app/components/shared/SkipLinks/SkipLink.tsx:1]: Should / can we push a SkipLink component back to ASC?

File: src/app/components/layouts/DefaultLayout/DefaultLayout.tsx [hunk 2]
```diff
@@ -24,9 +27,18 @@ const MenuWrap = styled.div`
   justify-content: space-between;
   width: 100%;
 `
+const BreadCrumbWrap = styled.div`
```
Comment (gilleswittenberg) [inline src/app/components/layouts/DefaultLayout/DefaultLayout.tsx:30]: This could not be solved with `<Row>` + `<Column>`?

**Review by gilleswittenberg — APPROVED** [review]

File: src/app/components/shared/SkipLinks/SkipLink.tsx (hunk 1 above)
Comment (remcohoff) [reply src/app/components/shared/SkipLinks/SkipLink.tsx:1]: Would be nice, but it has to be much more adjustable then.

File: src/app/components/layouts/DefaultLayout/DefaultLayout.tsx (hunk 2 above)
Comment (remcohoff) [reply src/app/components/layouts/DefaultLayout/DefaultLayout.tsx:30]: No, this css is for the centering on the page (like <main> does)

Comment (gilleswittenberg) [reply src/app/components/layouts/DefaultLayout/DefaultLayout.tsx:30]: Ok, let's move `BreadCrumbWrap` into its own file, then. Not cluttering `DefaultLayout` with it.

File: src/app/components/shared/SkipLinks/SkipLink.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    left: 50%;
+    padding: 12px 16px;
+    position: absolute;
+    transform: translateY(-100%);
+    transition: transform 0.3s;
+    z-index: -999;
+
+    &:focus {
+      transform: translateY(0%);
+      z-INDEX: 999;
```
Comment (gilleswittenberg) [inline src/app/components/shared/SkipLinks/SkipLink.tsx:18]: lowercase `z-index`

File: src/app/components/shared/SkipLinks/SkipLink.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components"
+
+const Nav = styled.nav`
+a {
+    background: #EC0000;
```
Comment (gilleswittenberg) [inline src/app/components/shared/SkipLinks/SkipLink.tsx:5]: You can use `themeColor()` for both these colors

**Pull request merged**